src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -162,19 +162,17 @@
                                           (reset! show-password-validation? true))}]]
 
          [rn/view {:style style/bottom-part}
+          [rn/view {:style style/disclaimer-container}
+           [quo/disclaimer
+            {:on-change #(reset! accepts-disclaimer? %)
+             :checked?  @accepts-disclaimer?}
+            (i18n/label :t/password-creation-disclaimer)]]
+
           (when (= @focused-input :password)
             [help
              {:validations       validations
               :password-strength password-strength}])
 
-          (when (= @focused-input :repeat-password)
```

Comment:
Hi @SilentCruzer 

Just a small comment:

This code was thought to be more general and sets a keyword id according to the input focused (those keywords are be `:password` and `:repeat-password`), but since now we don't care about the `:repeat-password` value, we could simplify that atom and logic. The callback responsible of setting this id in the atom is `on-input-focus`.

Would you mind making that simplification, please?